content/en/guides/internals-glossary/internals-module-container.md

```diff
@@ -55,6 +55,11 @@ This method returns final `{ dst, src, options }` object.
 
 ### addPlugin (template)
 
+- **template**: On top of [`addTemplate`](#addtemplate-template)'s `template` Object properties (`src`, `options`, and `fileName`),
+the parameter `template` in this scope also has plugin-specific options:
+  - `mode` (`'all'`, `'client'` or `'server'`): as in plugins [mode](/docs/2.x/directory-structure/plugins#object-syntax) 
+  - `ssr` (`Boolean`): Prefer `mode`, as this option will be deprecated in Nuxt 3.
```

Comment:
Neither ssr or mode options are not recommended. Mode is better be part of plugin name like `axios.client.js`

Reply:
Ah, good to know. I'll change the content, then ;) Thanks, Pooya!